core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -70,10 +71,17 @@ public static MetricsConfig fromProperties(Map<String, String> props) {
           }
           spec.columnModes.put(columnAlias, mode);
         });
-
```

Comment:
Nit: this is an unnecessary whitespace change that could lead to git conflicts. Could you add it back?